plugins/woocommerce/src/Admin/Features/OnboardingTasks/Tasks/StoreDetails.php

```diff
@@ -69,7 +69,7 @@ public function get_action_url() {
 	 * @return bool
 	 */
 	public function is_complete() {
-		$profiler_data = get_option( OnboardingProfile::DATA_OPTION, array() );
-		return isset( $profiler_data['completed'] ) && true === $profiler_data['completed'];
+		// Mark as completed if the store address and city are set. We don't need to check the country because it's set by default.
```

Comment:
@chihsuan I'm a bit concerned about having the completion logic changed, since I think it is possible that users may only complete the first step and navigate away from the onboarding wizard, and then they would have no way to come back since their address would be completed.

If the concern with using the `completed` option in profiler for WordPress.com, I think https://github.com/Automattic/wp-calypso/pull/63965 resolves it by setting the option to completed from the stepper.

What do you think?

Reply:
@ilyasfoo Good question! I think Veronica has answered a question one in the issue. 

_"How do users access Industry, Product Types, and Business Details again?"_

> Is it something users usually do? My assumption is that no one would go back to fill this data as they don't have any impact on setting up their store (apart from store location). They are mostly data that we use to profile them, but once skipped the profiler, there's no reason for our users to spend time selecting industry or type of products. But if we have any data that demonstrates the opposite, I'm happy to re-discuss the designs 

https://github.com/woocommerce/woocommerce/issues/32587#issuecomment-1123913127

Does that make sense to you?

Reply:
Got it, thank you for the context!